### PR TITLE
Add Upcoming length adjustment feature flag

### PR DESCRIPTION
**Pull request by SamBobBarnes**
Add feature flag for upcoming length adjustment feature.

### CODE REVIEW

File: packages/desktop-client/src/components/settings/Experimental.tsx [hunk 1]
```diff
@@ -116,6 +116,12 @@ export function ExperimentalFeatures() {
             >
               <Trans>Rule action templating</Trans>
             </FeatureToggle>
+            <FeatureToggle
+              flag="upcomingLengthAdjustment"
+              // feedbackLink="https://github.com/actualbudget/actual/issues/3606" TODO: Create feedback issue
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/settings/Experimental.tsx:121]: **💬 suggestion:** ‏created it for you :)
```suggestion
              feedbackLink="https://github.com/actualbudget/actual/issues/3660"
```

**Review by MatissJanis — COMMENTED** [review]
Code LGTM! Would you mind updating the VRT screenshots so the vrt CI job would pass?

**Comment by SamBobBarnes** [comment]
> Would you mind updating the VRT screenshots so the vrt CI job would pass?

@MatissJanis is there a way to generate these?

**Comment by youngcw** [comment]
> > Would you mind updating the VRT screenshots so the vrt CI job would pass?
> 
> @MatissJanis is there a way to generate these?

The readme for the desktop client package says how to generate new ones

**Comment by SamBobBarnes** [comment]
> The readme for the desktop client package says how to generate new ones

@MatissJanis I'm sure I'm doing something wrong, but I've been pulling my hair out for 2 days trying to get these VRT screenshots generated and getting nowhere.

Most common error I get when I try to run them is `net::ERR_CONNECTION_REFUSED at https://localhost:3100/`

I have tried to run them locally, in docker, and against localhost and my lan ip.
I am running the app via `yarn start` with HTTPS=true env variable set.
I can hit the site with my browser, but playwright is acting like it doesn't exist.
I have also tried to run the server via the `docker compose up --build` command, but that runs into an expired cert somewhere and doesnt build.

Could this be CORS acting up? 

I wish there was better documentation on how to get this working. Seems like the current docs in the readme dont have any good troubleshooting tips. Any helpful tips you could give would be greatly appreciated. I have not used playwright before, so could by my env as well. I have tried this on windows and mac with the same result.

**Comment by youngcw** [comment]
have you tried your actual IP instead of localhost?  I cant use localhost when inside the docker container

**Comment by youngcw** [comment]
My process is to:
* run yarn just to make sure all deps are met
* open a new terminal run `HTTPS=true yarn start`
* in original terminal run `yarn vrt:docker --e2e-start-url https://192.168.1.3:3001 --update-snapshots`

I've never been able to use localhost and have it work so maybe just don't even try using localhost in place of the IP.  I just filled in an example IP for now.  Then git status should show the updated snapshots if any needed changed

**Comment by SamBobBarnes** [comment]
> My process is to:
> 
> * run yarn just to make sure all deps are met
> * open a new terminal run `HTTPS=true yarn start`
> * in original terminal run `yarn vrt:docker --e2e-start-url https://192.168.1.3:3001 --update-snapshots`
> 
> I've never been able to use localhost and have it work so maybe just don't even try using localhost in place of the IP. I just filled in an example IP for now. Then git status should show the updated snapshots if any needed changed

Tried it as described on my mac, and it worked first time. Dont know what I was doing wrong other than localhost. Thanks @youngcw!

**Review by joel-jeremy — APPROVED** [review]

**Pull request merged**